doc/appendix-at-rksv/function-structures/function-structures.md

```diff
@@ -1,6 +1,6 @@
 ## Function Structures
 
-This chapter expands on the descriptions of the function structures covered in the Chapter ["Function structures"](../../general/function-structures/function-structures.md) of the General Part, with country specific information applicable to the Austrian market.
+This chapter expands on the descriptions of the function structures covered in the Chapter ["Function structures"](../../general/function-structures/function-structures.md) of the General Part, with country-specific information applicable to the Austrian market.
```

Comment:
More detailed descriptions of the function structures covered in Chapter bla can be found here
(so it is not monotonous)

Reply:
;-)